CONTRIBUTING.md

```diff
@@ -6,6 +6,41 @@ Please create any new issue in the most appropriate repository (see full list on
 
 Eclipse Bugzilla issues are deprecated for Xtext. However, we still accept them.
 
+### Milestones
+
+We have one milestone for each Xtext release. Fixed issues should be assigned to the milestone of the nearest release in which the fix will be included. For example, if an issue is fixed on the `maintenance` branch, the corresponding milestone is the next service release (increasing the third version component). If an issue is fixed on the `master` branch, the next major or minor release should be assigned.
+
+It is possible to assign issues to a milestone before they are fixed, which can be useful for planning. However, make sure someone will actually work on that issue when you do that!
```

Comment:
As written in the proposal, you can use the milestones to do some release planning. We should just avoid piling up too many issues in a milestone, but instead make sure the planning can be realized by the committers who are currently available. In the past we had the Bugzilla whiteboard for this kind of planning, and there we had too many issues that were assigned to a release but couldn't be solved, so they were pushed to the next release.